styles.css

```diff
@@ -511,13 +502,65 @@ hr {
   background-color: #0d151e;
 }
 
+.see-more .see {
+  text-decoration: none;
+}
+
+.mobile-project-container {
+  display: none;
+  z-index: 10;
+  position: fixed;
+  background: rgba(106, 110, 116, 0.7);
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  top: 0;
+  padding-left: 30px;
+}
+
+.popup-item-mob {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  background: #fff;
+  margin-top: 20px;
+  border-right-width: 20px;
+  margin-left: 20px;
+  margin-bottom: 20px;
+  border-radius: 10px;
+  padding: 15px;
+}
```

Comment:
- [ ] Nice implementing the design, however, the pop-up window is not centered correctly  in the mobile version so please it would be better if you fix that 
![image](https://user-images.githubusercontent.com/61790587/169364632-2d12af1f-4aed-49cf-ad73-b09dc5f1043d.png)